web/src/features/fireWeather/components/tables/ComparisonTableRow.tsx

```diff
@@ -207,218 +218,221 @@ const formatAccumulatedPrecipitation = (
 const ComparisonTableRow = (props: Props) => {
   const classes = useStyles()
 
-  console.log(props.headers)
-
   return (
     <TableRow data-testid={props.testId}>
       {props.index}
       {props.headers.map((variable: WeatherVariable, idx: number) => {
-          return props.subheaders[idx].map((source: DataSource) => {
-            let cellContent = undefined
-            switch (source) {
-              case DataSource.Observed: {
-                switch (variable) {
-                  case WeatherVariable.Temperature: {
-                    cellContent = formatTemperature(props.observation)
-                    break
+        const colStyle = idx % 2 === 0 ? classes.darkColumn : classes.lightColumn
+        return props.subheaders[idx].map((source: DataSource) => {
+          let cellContent = undefined
+          switch (source) {
```

Comment:
Lots of cyclomatic complexity with all these switch statements (I'd argue we should turn this on as a lint check: http://pylint.pycqa.org/en/latest/technical_reference/extensions.html#design-checker)

A simpler solution I'd propose is creating a map of `[WeatherVariable, formatFn]` and just lookup the function to call with the props and class. Lmk if you want more of an explanation.